tonic/src/types.rs

```diff
@@ -0,0 +1,182 @@
+use prost::{Message, Name};
+
+use celestia_types::auth::{AuthParams, BaseAccount};
+use celestia_types::blob::BlobParams;
+
+use celestia_proto::celestia::blob::v1::{
+    QueryParamsRequest as QueryBlobParamsRequest, QueryParamsResponse as QueryBlobParamsResponse,
+};
+use celestia_proto::cosmos::auth::v1beta1::{
+    BaseAccount as RawBaseAccount, QueryAccountRequest, QueryAccountResponse, QueryAccountsRequest,
+    QueryAccountsResponse,
+};
+use celestia_proto::cosmos::auth::v1beta1::{
+    QueryParamsRequest as QueryAuthParamsRequest, QueryParamsResponse as QueryAuthParamsResponse,
+};
+use celestia_proto::cosmos::base::node::v1beta1::{ConfigRequest, ConfigResponse};
+use celestia_proto::cosmos::base::tendermint::v1beta1::{
+    GetBlockByHeightRequest, GetBlockByHeightResponse, GetLatestBlockRequest,
+    GetLatestBlockResponse,
+};
+use celestia_proto::cosmos::tx::v1beta1::{BroadcastMode, BroadcastTxRequest, GetTxRequest};
+
+use celestia_tendermint::block::Block as TendermintBlock;
+
+use celestia_tendermint_proto::v0_34::types::BlobTx;
+
+use cosmrs::crypto::PublicKey;
+use cosmrs::Any;
+
+use crate::Error;
+
+pub mod tx;
+
+pub struct Block(pub TendermintBlock);
```

Comment:
why do we need a newtype for Block?

Reply:
Actually we don't strictly need it, removed.